src/proxyWorker.js

```diff
@@ -77,8 +77,8 @@ function EventListener() {
     list.splice(me, 1);
   };
 
-  this.fireEvent = function fireEvent(event) {
-    event.preventDefault = function(){};
+  this.fireEvent = (event) => {
+    event.preventDefault = () => {};
 
     if (event.type in this.listeners) {
```

Comment:
`this` is different after this change, isn't it? But I suspect it was wrong before?

Reply:
Thanks .. I missed that one!